.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,70 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # Tests on Windows elicit errors so keeping only ubuntu/macos for now
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.6, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Display Python version
+      run: python -c "import sys; print(sys.version)"
+
+    - name: Install package
+      run: pip install .[tools,test]
+
+    - name: Lint with flake8
```

Comment:
So we are using pep8 as coding style ? We should discuss it at some point

Reply:
Yes, I think you also didn't like Black, so for now pep8 should be ok, no?

Reply:
Yes seems good to me .

Reply:
+1 for PEP8, also coz it's well spread in neuro-softwares...